FutureNorthants/northants-design-system
# Instagram icon / added to footer

### Testing

1. View the new Instagram icon in the Icons section
2. View the footer structure and verify the Instagram icon is in there, linked to the appropriate site dependent upon theme selection

### Deploy notes
This is currently branched off the homepage promo branch, will need to be against master if that work goes out before this does.
